db/migrate/20231031213740_add_effect_type_to_alchemical_properties.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddEffectTypeToAlchemicalProperties < ActiveRecord::Migration[7.1]
+  def change
+    add_column :alchemical_properties, :effect_type, :string
```

Comment:
As mentioned in "Considerations", we ultimately want this column `NOT NULL`, but have to backfill data before we can do that.